app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts

```diff
@@ -136,28 +137,31 @@ export class LocationInputComponent implements OnInit {
   }
 
   get latitudeField(): FormConfigField {
-    if (this.object.isSpaceGeom) {
+    if (this.object.isSpaceGeom && this.object.isSpaceGeom.embeddedFields) {
```

Comment:
embeddedFields are prop of object not isSpaceGeom, that is boolean

Reply:
fixed